src/main/java/com/bookstore/entity/Book.java

```diff
@@ -0,0 +1,34 @@
+package com.bookstore.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
It should be done by previous HW

```suggestion
@Getter
@Setter
@Table....
```